packages/data-store/src/types/digitalCredential/digitalCredential.ts

```diff
@@ -93,6 +93,7 @@ export enum CredentialRole {
   ISSUER = 'ISSUER',
   VERIFIER = 'VERIFIER',
   HOLDER = 'HOLDER',
+  FEDERATION_OPERATOR = 'FEDERATION OPERATOR'
```

Comment:
FEDERATION_OPERATOR instead of FEDERATION OPERATOR (also in the SQL above,  This might cause issues in some SQL implementations.)

Reply:
I also understood that enum keys & values should be the same to allow casting.

Reply:
ye changed it